src/devtools/client/debugger/src/components/shared/ResultList.tsx

```diff
@@ -61,7 +60,7 @@ export default class ResultList extends Component<ResultListProps> {
     };
 
     return (
-      <li {...props}>
+      <li key={`${item.id}${item.value}${index}`} {...props}>
```

Comment:
Curious why `item.id` isn't sufficiently unique for the key here?

Reply:
Not sure. That key construction was there previously, I think.

Reply:
Just looks unnecessarily complicated :)